README.md

```diff
@@ -1,16 +1,6 @@
 # Getting Started with the Vue Tree Grid
 
-A quick-start project that shows how to work with the Syncfusion Tree Grid in a Vue application. This project contains simple code customization, as well as some important features like binding data, customizing columns, and enable paging, sorting, and filtering.
-
-Refer to the following documentation for the Syncfusion Vue Tree Grid component: 
-https://ej2.syncfusion.com/vue/documentation/treegrid/getting-started-vue-3/
```

Comment:
No need to delete this link since it is not getting start stand but getting started 3

Reply:
Okay